src/components/giphy/searchBar.js

```diff
@@ -0,0 +1,56 @@
+
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 30%;
+  transform: translate(-50%, -50%);
+`
+
+const SearchContainer = styled.div`
+  width: 100%;
+  margin-left: 100px;
+  margin-bottom: -40px;
+  display: flex;
+`
+const Input = styled.input`
+  width: 100%;
+  border: 3px solid #00B4CC;
+  border-right: none;
+  padding: 5px;
+  height: 20px;
+  border-radius: 5px 0 0 5px;
+  outline: none;
+  color: #9DBFAF;
+`
+
+const Button = styled.button`
+  width: 100px;
+  height: 36px;
+  border: 1px solid #00B4CC;
+  background: #00B4CC;
+  text-align: center;
+  color: #fff;
+  border-radius: 0 5px 5px 0;
+  cursor: pointer;
+  font-size: 20px;
+`
+
+function SearchBar(props) {
+  const [searchData, setSearchData] = useState(0);
```

Comment:
```suggestion
  const [searchData, setSearchData] = useState("");
```
nó là string thì mình nên để là "" nha e, array thì [], nào number thì mới để là 0